specification/cognitiveservices/data-plane/Luis/v2.0/LUIS Programmatic APIs v2.0.swagger.json

```diff
@@ -0,0 +1,5225 @@
+        "operationId": "examples - Batch add labels",
+        "parameters": [
+          {
+            "name": "appId",
+            "in": "path",
+            "description": "Format - guid. The application ID.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "versionId",
+            "in": "path",
+            "description": "The task version ID.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "exampleLabelObjectArray",
+            "in": "body",
+            "schema": {
+              "$ref": "#/definitions/ExampleLabelObjectArray"
+            },
+            "description": "A JSON array containing example labels."
+          }
+        ],
+        "consumes": [
+          "application/json"
+        ],
+        "responses": {
+          "201": {
+            "description": "A string array determining which labels were added successfully.",
+            "examples": {
+              "application/json": "[  {    \"value\": {      \"UtteranceText\": \"book me a flight from cairo to redmond next thursday\",      \"ExampleId\": -11    },    \"hasError\": false  },  {    \"value\": {      \"UtteranceText\": \"what's the weather like in seattle?\",      \"ExampleId\": -12    },    \"hasError\": false  }]"
+            }
+          }
+        },
+        "produces": [
+          "application/json"
+        ]
+      },
+      "get": {
+        "description": "Returns examples to be reviewed.",
+        "operationId": "examples - Review labeled examples",
+        "parameters": [
+          {
+            "name": "appId",
+            "in": "path",
+            "description": "Format - guid. The application ID.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "versionId",
+            "in": "path",
+            "description": "The task version ID.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "skip",
+            "in": "query",
+            "description": "The number of entries to skip. Default value is 0.",
+            "type": "integer",
+            "default": 0,
+            "x-nullable": "false",
+            "x-ms-enum": {
+              "name": "SkipParameter",
+              "modelAsString": false
+            },
+            "enum": [
+              0
+            ]
+          },
+          {
+            "name": "take",
+            "in": "query",
+            "description": "The number of entries to return. Maximum page size is 500. Default is 100.",
+            "type": "integer",
+            "default": 100,
+            "x-nullable": "false",
+            "x-ms-enum": {
+              "name": "TakeParameter",
+              "modelAsString": false
+            },
+            "enum": [
+              100
+            ]
+          }
+        ],
+        "responses": {
+          "200": {
```

Comment:
is this a pageable operaation? see https://github.com/Azure/autorest/blob/master/docs/extensions/readme.md#x-ms-pageable